include/IECoreScene/TypedObjectParameter.h

```diff
@@ -42,35 +42,20 @@ namespace IECoreScene
 
 IE_CORE_FORWARDDECLARE( Renderable )
 IE_CORE_FORWARDDECLARE( StateRenderable )
```

Comment:
Should this and the other references to StateRenderable in this file also go as part of the StateRenderable removal?

Reply:
And likewise for MotionPrimitive in `include/IECoreScene/TypedPrimitiveParameter.h`?

Reply:
Thanks for catching those - fixups in https://github.com/ImageEngine/cortex/pull/1477/commits/e072080502bda40f4f23a3e3b29ae6b6f80b7048 and https://github.com/ImageEngine/cortex/pull/1477/commits/1b35ec609ebbe860be5eebf3b8a9507e1bd3a00d.

Reply:
Thanks!